examples/03-tips-n-tricks/00-example-template.py

```diff
@@ -0,0 +1,105 @@
+"""
+.. _ref_how_to_add_an_example_reference_key:
+
+How to add a new example
+------------------------
+This example demonstrates how to add new examples as well as being a template that can
+be used in their creation.
+
+This block comment should be included at the top of any new example. Each example
+should have a reference tag/key in the form:
+
+``.. _ref_my_example:``
+
+The ``.. _ref_`` is necessary. Everything that follows is your reference tag. As
+convention, we keep all references all in ``snake_case``.
+
+This section should give a brief overview of what the example is about and/or demonstrates.
+The title should be changed to reflect the topic your example covers.
+
+New examples should be added as python scripts to:
+
+``PyMAPDL/examples/XY-example-folder-name/``
+
+Avoid creating new folders unless absolutely necessary. If in doubt put the example
+in the folder closest to what it is doing and its precise location can be advised
+on in the pull request! If you *must* create a new folder, make sure to add a
+``README.txt`` containing a reference, a title and a single sentence description of the folder.
+Otherwise the new folder will be ignored by Sphinx!
```

Comment:
See: https://developers.google.com/style/exclamation-points

```suggestion
.. note::
   Avoid creating new folders unless absolutely necessary. If in doubt put the example
   in the folder closest to what it is doing and its precise location can be advised
   on in the pull request. If you *must* create a new folder, make sure to add a
   ``README.txt`` containing a reference, a title and a single sentence description of the folder.
   Otherwise the new folder will be ignored by Sphinx.
```